UmbracoAuthorizedController cleanup

Migaroez opened the pull request:
## Description

Removed mentioning of UmbracoAuthorizedController as it should no longer be used in v10 and up

## Type of suggestion

* [ ] Typo/grammar fix
* [x] Updated outdated content
* [ ] New content
* [ ] Updates related to a new version
* [ ] Other

## Product & version (if relevant)
v10 and up

alina-tincas commented on 10/umbraco-cms/implementation/controllers.md:
```diff
@@ -52,8 +52,34 @@ An Umbraco Authorized Controller is used when the controller requires member or
 The Umbraco Authorized controllers and attributes for Backoffice Users are:
 
 *   **MVC**
+    There is no specific controller available to inherit from. We recommend inheriting from the basic `Umbraco.Cms.Web.Common.Controllers.UmbracController`` and applying the following attributes to your method
```
checking https://apidocs.umbraco.com/v13/csharp/api/Umbraco.Cms.Web.Common.Controllers.UmbracoController.html it seems to be `Umbraco.Cms.Web.Common.Controllers` instead of `Umbraco.Cms.Web.Common.Controllers.UmbracController` can you please confirm as well? 😊 
```suggestion
    There is no specific controller available to inherit from. We recommend inheriting from the basic `Umbraco.Cms.Web.Common.Controllers.Controllers` and applying the following attributes to your method:
```

alina-tincas commented on 10/umbraco-cms/implementation/controllers.md:
```diff
@@ -52,8 +52,34 @@ An Umbraco Authorized Controller is used when the controller requires member or
 
 *   **MVC**
+    There is no specific controller available to inherit from. We recommend inheriting from the basic `Umbraco.Cms.Web.Common.Controllers.UmbracController`` and applying the following attributes to your method
+    - `[Authorize(Policy = AuthorizationPolicies.BackOfficeAccess)]`: Uses .net authorization using the BackOfficeAccess policy. We recommend adding extra custom authorization policies for your endpoints.
```
```suggestion
    - `[Authorize(Policy = AuthorizationPolicies.BackOfficeAccess)]`: Uses .NET authorization using the BackOfficeAccess policy. We recommend adding extra custom authorization policies for your endpoints.
```

alina-tincas commented on 10/umbraco-cms/implementation/controllers.md:
```diff
@@ -52,8 +52,34 @@ An Umbraco Authorized Controller is used when the controller requires member or
-    A base class implementation for backoffice authorized controllers is inherited from: `Umbraco.Cms.Web.Common.Controllers.UmbracoAuthorizedController`. These MVC controllers are not auto-routed.
+    Remember to [add a route](../reference/routing/authorized/defining-a-route) for your controller
+
+    ###### Example custom authorized backoffice mvc controller
```
previous sections has 3 ### so as this is an example part of same section it should have only 4 ####
```suggestion
    #### Example custom authorized backoffice MVC Controller
```

alina-tincas commented on 10/umbraco-cms/implementation/controllers.md:
```diff
@@ -52,8 +52,34 @@ An Umbraco Authorized Controller is used when the controller requires member or
+    - `[DisableBrowserCache]`: Tells the browser to not cache the result.
 
-    A base class implementation for backoffice authorized controllers is inherited from: `Umbraco.Cms.Web.Common.Controllers.UmbracoAuthorizedController`. These MVC controllers are not auto-routed.
+    Remember to [add a route](../reference/routing/authorized/defining-a-route) for your controller
```
```suggestion
    Remember to [add a route](../reference/routing/authorized/defining-a-route) for your controller.
```

alina-tincas reviewed (changes requested):
Hi @Migaroez thank you for the PR 🙌 

I have 1 suggestions regarding the class, can you please look into it and confirm if this is the correct one? 😊

Migaroez commented:
hey @alina-tincas Thx for the suggestions.
- fixed the namespaced class name
- added your punctuation/casing changes to all documents
- removed code that i forgot to remove from 2 verions.

alina-tincas commented on 10/umbraco-cms/implementation/controllers.md:
```diff
@@ -52,8 +52,34 @@ An Umbraco Authorized Controller is used when the controller requires member or
-    A base class implementation for backoffice authorized controllers is inherited from: `Umbraco.Cms.Web.Common.Controllers.UmbracoAuthorizedController`. These MVC controllers are not auto-routed.
+    Remember to [add a route](../reference/routing/authorized/defining-a-route) for your controller.
+
+    #### Example custom authorized backoffice mvc controller
```
```suggestion
    #### Example custom authorized backoffice MVC Controller
```

alina-tincas commented on 12/umbraco-cms/reference/routing/authorized.md:
```diff
@@ -24,13 +24,12 @@ The {area} specified by the [PluginController] attribute replaces the /api/ area
 ## MVC controllers for the backoffice
 
-If you are using MVC in the backoffice then you would normally inherit from `Umbraco.Cms.Web.Common.Controllers.UmbracoAuthorizedController`. This type of controller is not auto-routed like Umbraco Api controllers or surface controllers, so will require a custom route declaration and be registered with the Umbraco DI container to make it work.
+Depending on the type of controller used (mvc or webapi), the controller is not auto-routed. You will need to delcare a custom route and register it with the Umbraco DI container to make it work.
```
```suggestion
Depending on the type of controller used (MVC or WebAPI), the controller is not auto-routed. You will need to declare a custom route and register it with the Umbraco DI container to make it work.
```

alina-tincas reviewed (approved):
small changes

alina-tincas commented:
Thank you for making the changes @Migaroez #h5YR 

All looks good now, so am merging this in 😊
